zcash_client_sqlite/src/wallet.rs

```diff
@@ -1339,7 +1339,10 @@ pub(crate) fn get_unspent_transparent_outputs<P: consensus::Parameters>(
 
         let index: u32 = row.get(1)?;
         let script_pubkey = Script(row.get(2)?);
-        let value = Amount::from_i64(row.get(3)?).unwrap();
+        let value_raw: i64 = row.get(3)?;
```

Comment:
`wallet/sapling.rs` line 103 does the same thing without the temporary `value_raw` variable; I think that's just as clear and more concise.